backend/src/test/java/com/soen341/instagram/service/CommentServiceTest.java

```diff
@@ -42,7 +42,7 @@
 	private final static long pictureId = 1;
 	private final static String validComment = "This is a valid comment";
```

Comment:
It just occurred to me that these are all constants. Did we previously establish a standard for naming them?

Reply:
Usually constants are always upper case and separated by underscores. I think we should go for that

Reply:
agree, I'll make the switch to capitals